README.md

```diff
@@ -1388,8 +1388,8 @@ arXiv:2003.00636, 2020.
 *[Event-based Asynchronous Sparse Convolutional Networks](https://arxiv.org/pdf/2003.09148)*,  
 European Conf. Computer Vision (ECCV), 2020.
 - <a name="Hu20arxiv"></a>Hu, Y., Delbruck, T., Liu, S.-C.,  
-*[Exploiting Event Cameras by Using a Network Grafting Algorithm](https://arxiv.org/pdf/2003.10959)*,  
-arXiv:2003.10959, 2020.
+*[Learning to Exploit Multiple Vision Modalities by Using Grafted Networks](https://arxiv.org/pdf/2003.10959)*,  
+European Conference on Computer Vision (ECCV), Online, 2020.
```

Comment:
No "Online", please. In general, we do not specify locations of conferences. Check other ECCV entries

Reply:
Thanks! I've changed it so the item now aligns with other entries